### PR TITLE
media-gfx/tuxpaint-0.2.8 and media-gfx/tuxpaint-stamps-20220604

### PR DESCRIPTION
**Pull request by rndxelement**
I added an ebuild for media-gfx/tuxpaint-0.2.8.
I runtime tested a few of the new features and they seem to work fine ([changelog](https://tuxpaint.org/docs/CHANGES.txt)).

Also added an ebuild for media-gfx/tuxpaint-stamps-20220604 and tested the stamps in tuxpaint (they seem to work).

Furthermore, I added myself as a proxied maintainer for media-gfx/tuxpaint-stamps, because I already proxy maintain media-gfx/tuxpaint.

### CODE REVIEW

File: media-gfx/tuxpaint-stamps/tuxpaint-stamps-20220604.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P=${PN}-${PV:0:4}.${PV:4:2}.${PV:6:2}
+
+DESCRIPTION="Set of 'Rubber Stamp' images which can be used within Tux Paint"
+HOMEPAGE="http://www.tuxpaint.org/"
```
Comment (juippis) [inline media-gfx/tuxpaint-stamps/tuxpaint-stamps-20220604.ebuild:9]: `https` here. Isn't there a secion in that site that's more accurate to for package?

File: media-gfx/tuxpaint-stamps/tuxpaint-stamps-20220604.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-20211125-make-382.patch
+)
+
+src_install() {
+	emake PREFIX="${D}/usr" install-all
+
+	rm -f docs/COPYING.txt || die
```
Comment (juippis) [inline media-gfx/tuxpaint-stamps/tuxpaint-stamps-20220604.ebuild:27]: Drop the `-f`. You should know what you're removing :P

File: media-gfx/tuxpaint/tuxpaint-0.9.28.ebuild [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+inherit desktop optfeature toolchain-funcs xdg
+
+DESCRIPTION="Drawing program designed for young children"
+HOMEPAGE="http://www.tuxpaint.org/"
+SRC_URI="mirror://sourceforge/${PN}/${P}-sdl1.tar.gz -> ${P}.tar.gz"
+S=${WORKDIR}/${P}-sdl1
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ppc x86"
```
Comment (juippis) [inline media-gfx/tuxpaint/tuxpaint-0.9.28.ebuild:15]: Don't commit straight to stable. Use `ekeyword ~all tuxpaint-0.9.28.ebuild` when bumping the ebuild.

File: media-gfx/tuxpaint/tuxpaint-0.9.28.ebuild [hunk 4]
```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop optfeature toolchain-funcs xdg
+
+DESCRIPTION="Drawing program designed for young children"
+HOMEPAGE="http://www.tuxpaint.org/"
```
Comment (juippis) [inline media-gfx/tuxpaint/tuxpaint-0.9.28.ebuild:9]: `https`

File: media-gfx/tuxpaint/tuxpaint-0.9.28.ebuild [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+	media-gfx/graphicsmagick[jpeg,png,svg]
+	sys-devel/gettext
+"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-Makefile.patch
+)
+
+src_compile() {
+	# parallel build may break things
```
Comment (juippis) [inline media-gfx/tuxpaint/tuxpaint-0.9.28.ebuild:44]: Is there a bug open about this that we could track if this gets fixed? Preferably upstream, but bugs.gentoo.org #id is fine too.

File: media-gfx/tuxpaint/tuxpaint-0.9.28.ebuild [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+	>=media-libs/libpng-1.2:0=
+	>=media-libs/freetype-2:2
+	media-libs/libsdl[joystick]
+	media-libs/sdl-image[png]
+	media-libs/sdl-mixer
+	media-libs/sdl-pango
+	media-libs/sdl-ttf
+	media-libs/sdl-gfx
+	media-gfx/libimagequant
+	x11-libs/cairo
```
Comment (juippis) [inline media-gfx/tuxpaint/tuxpaint-0.9.28.ebuild:30]:
```suggestion
	media-gfx/libimagequant
	sys-libs/zlib
	x11-libs/cairo
```

File: media-gfx/tuxpaint/tuxpaint-0.9.28.ebuild [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+	dev-libs/fribidi
+	gnome-base/librsvg:2
+	>=media-libs/libpng-1.2:0=
+	>=media-libs/freetype-2:2
+	media-libs/libsdl[joystick]
+	media-libs/sdl-image[png]
+	media-libs/sdl-mixer
+	media-libs/sdl-pango
+	media-libs/sdl-ttf
+	media-libs/sdl-gfx
```
Comment (juippis) [inline media-gfx/tuxpaint/tuxpaint-0.9.28.ebuild:28]:
```suggestion
	media-libs/sdl-gfx:=
```

**Review by juippis — COMMENTED** [review]
```
 * QA Notice: The ebuild is installing to one or more unexpected paths:
 * 
 *   /usr/share/doc/tuxpaint-dev
 * 
 * Please fix the ebuild to use correct FHS/Gentoo policy paths.
```
It should be */usr/share/doc/${PF}*, 
https://devmanual.gentoo.org/ebuild-writing/variables/index.html

**Comment by rndxelement** [comment]
@juippis Thanks for the review! I fixed the things you pointed out.
I actually couldn't find any references to the parallel make issue. I opened a [bug](https://bugs.gentoo.org/859169) for it and I'll try to pin point the reason for it soon.

**Review by juippis — APPROVED** [review]
Thanks!

**Pull request closed without merging**